### ISSUE TITLE
report.py seems to be broken

### ISSUE THREAD

**Issue by rfriese** [grid-control/grid-control#20]
Hi,

I have long tails in my grid-control runs and would like to examine what is happening. Helpful in this case is usually report.py. Unfortunately, it's broken atm. Even report.py -L does not work atm. The only report I get is the TimeReport, which is already very important but not the only information I need. I e.g. would like to know while the job is running, how long each dataset takes and so on.

cheers,
Raphael

**Comment by FredStober** [comment]
Getting the list of report plugins via `scripts/report.py -L` is indeed not working properly. However that list (with a nicer output) can be retrieved with `scripts/plugins.py Report`. I just tested `report.py -R module -J TODO -T` and that worked fine. Event rates are currently only calculated by the `plot` report.

**Comment by rfriese** [comment]
Hi Fred,
thanks a lot for pointing me to report.py.

the -T option does not work for me, I get the following error:
```

./scripts/report.py /nfs/dust/cms/user/rfriese/htautau/artus/2016-08-03_15-07_SvFit2015v3/grid-control_config.conf -R plot -T    

Exception occured in grid-control [1913]

PluginError: Error while creating instance: 'grid_control_cms.cmssw.CMSSW'
  APIError: Unable to convert default object: ''
    ConfigError: It is *not* possible to change "<cmssw cmssw> dataset provider" from 'ListProvider' to 'DBS3Provider'

In case this is caused by a bug, please send the log file:
	"/afs/desy.de/user/r/rfriese/xxl/analysis/CMSSW_7_1_5/src/grid-control/debug.log"
to grid-control-dev@googlegroups.com

```

Re-starting go.py on this config file is no problem. Do you know how I can get this running?

**Comment by FredStober** [comment]
Have you tried to use the work.conf in the work directory?

**Comment by rfriese** [comment]
same problem:

```
report.py /nfs/dust/cms/user/rfriese/htautau/artus/2016-08-05_11-11_SvFit2015v6/workdir/work.conf -R TimeReport -T

Exception occured in grid-control [1913]


PluginError: Error while creating instance: 'grid_control_cms.cmssw.CMSSW'
  APIError: Unable to convert default object: ''
    ConfigError: It is *not* possible to change "<cmssw cmssw> dataset provider" from 'ListProvider' to 'DBS3Provider'

In case this is caused by a bug, please send the log file:
	"/afs/desy.de/user/r/rfriese/xxl/analysis/CMSSW_7_1_5/src/grid-control/debug.log"
to grid-control-dev@googlegroups.com

```

**Comment by FredStober** [comment]
Fixed in 29d5c4ed68822351868766e884fd0d55c7644add (r1936).
